src/main/java/IfElse/PhraseAnalyser.java

```diff
@@ -0,0 +1,10 @@
+package IfElse;
```

Comment:
Программа должна анализировать текст и возвращать его описание в соответствии со следующими правилами:

"It stands no chance", если строка начинается со слов "Make" и заканчивается на "great again";
"It could be worse", если строка начинается со слов "Make" или заканчивается на "great again";
"It is fine, really", если строка не подходит под критерии выше;